whipper/common/accurip.py

```diff
@@ -151,33 +146,13 @@ def _download_entry(path):
     return resp.content
 
 
-def _save_entry(raw_entry, path):
-    logger.debug('saving AccurateRip entry to %s', path)
-    # XXX: os.makedirs(exist_ok=True) in py3
-    try:
-        makedirs(dirname(path))
-    except OSError as e:
-        if e.errno != EEXIST:
-            logger.error('could not save entry to %s: %r' % (path, str(e)))
-            return
-    open(path, 'wb').write(raw_entry)
-
-
 def get_db_entry(path):
     """
-    Retrieve cached AccurateRip disc entry as array of _AccurateRipResponses.
-    Downloads entry from accuraterip.com on cache fault.
+    Downloads entry from accuraterip.com.
```

Comment:
Docstrings are generally written in the imperative form, and this one was before too. Maybe:
```suggestion
    Download entry from accuraterip.com.
```